segment.go

```diff
@@ -369,11 +369,14 @@ func (seg *segment) lookupByOff(slot []entryPtr, hash16 uint16, offset int64) (i
 }
 
 func (seg *segment) resetStatistics() {
-	atomic.StoreInt64(&seg.totalEvacuate, 0)
-	atomic.StoreInt64(&seg.totalExpired, 0)
-	atomic.StoreInt64(&seg.overwrites, 0)
+	atomic.StoreInt64(&seg.entryCount, 0)
 	atomic.StoreInt64(&seg.hitCount, 0)
 	atomic.StoreInt64(&seg.missCount, 0)
+	atomic.StoreInt64(&seg.overwrites, 0)
+	atomic.StoreInt64(&seg.totalCount, 0)
+	atomic.StoreInt64(&seg.totalTime, 0)
```

Comment:
totalTime and totalCount is not stats.

Reply:
When `Cache.ResetStatistics` is called, we don't want to change them.

Reply:
Ok, thanks for pointing that out. I will follow up with a fix.